apps/verticalhome/style/css/app.css

```diff
@@ -22,6 +30,28 @@ gaia-grid {
   transition: height 0s 0.5s;
 }
 
+gaia-grid-rs .icon {
+  background-position: center !important;
+  background-size: 50px !important;
+  transition: all 0.5s ease 0s;
```

Comment:
"!important" modifier is needed, because .icon:not(.active) overlaps it and animation does not work.